### PR TITLE
Fix wav player warning

### PR DESCRIPTION
**Pull request by NoSloppy**
This seems to do the trick.

### CODE REVIEW

File: props/saber_BC_buttons.h [hunk 1]
```diff
@@ -714,24 +715,22 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
 // Spoken Battery Level in volts
     case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
       if (!mode_volume_) {
-        sound_library_.SayBatteryLevel();
+        sound_library_.SayTheBatteryLevelIs();
```
Comment (profezzorn) [inline props/saber_BC_buttons.h:718]: This seems unrelated to this change.

File: props/saber_BC_buttons.h [hunk 2]
```diff
@@ -356,6 +356,7 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     DetectTwist();
     Vec3 mss = fusor.mss();
     sound_library_.Poll(wav_player);
+    if (!wav_player) wav_player.Free();
```
Comment (profezzorn) [inline props/saber_BC_buttons.h:359]: I don't think this does anything. If wav_player is false, then there is no need to free it..

I think what you want is actually something like:

if (wav_player && !wav_player->IsPlaying()) wav_player.Free();

**Comment by NoSloppy** [comment]
Resubmitting alone
